target/linux/mvebu/files-6.6/arch/arm64/boot/dts/marvell/armada-7040-v-80.dts

```diff
@@ -0,0 +1,425 @@
+		marvell,pins = "mpp12";
+		marvell,function = "gpio";
+	};
+
+	pmx_gpio_leds_pins: gpio-leds-pins {
+		marvell,pins = "mpp13", "mpp23", "mpp26", "mpp33", "mpp34", "mpp51",
+			       "mpp52", "mpp54";
+		marvell,function = "gpio";
+	};
+
+	pmx_reg_usb_vbus_pins: reg-usb-vbus-pins {
+		marvell,pins = "mpp24";
+		marvell,function = "gpio";
+	};
+
+	pmx_board_id_pins: board-id-pins {
+		marvell,pins = "mpp4", "mpp10", "mpp62";
+		marvell,function = "gpio";
+	};
+
+	pmx_eeprom_wp_pins: eeprom-wp-pins {
+		marvell,pins = "mpp55";
+		marvell,function = "gpio";
+	};
+};
+
+&cp0_mdio {
+	status = "okay";
+
+	pinctrl-0 = <&pmx_cp0_mdio_pins>;
+	pinctrl-names = "default";
+
+	/* Marvell 88E6352 */
+	switch@0 {
+		compatible = "marvell,mv88e6085";
+		reg = <0x0>;
+
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			swport0: port@0 {
+				reg = <0x0>;
+				label = "lan5";
+				phy-handle = <&swphy0>;
+			};
+
+			swport1: port@1 {
+				reg = <0x1>;
+				label = "lan4";
+				phy-handle = <&swphy1>;
+			};
+
+			swport2: port@2 {
+				reg = <0x2>;
+				label = "lan3";
+				phy-handle = <&swphy2>;
+			};
+
+			swport3: port@3 {
+				reg = <0x3>;
+				label = "lan2";
+				phy-handle = <&swphy3>;
+			};
+
+			swport4: port@4 {
+				reg = <0x4>;
+				label = "lan1";
+				phy-handle = <&swphy4>;
+			};
+
+			port@5 {
+				reg = <0x5>;
+				ethernet = <&cp0_eth0>;
+				phy-connection-type = "sgmii";
+
+				fixed-link {
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+		};
+
+		mdio {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			swphy0: switch-phy@0 {
+				reg = <0x0>;
+			};
+
+			swphy1: switch-phy@1 {
+				reg = <0x1>;
+			};
+
+			swphy2: switch-phy@2 {
+				reg = <0x2>;
+			};
+
+			swphy3: switch-phy@3 {
+				reg = <0x3>;
+			};
+
+			swphy4: switch-phy@4 {
+				reg = <0x4>;
+			};
+		};
+	};
+
+	/* Marvell 88E1512 */
+	ethphy1: ethernet-phy@1 {
+		compatible = "ethernet-phy-id0141,0dd1",
+			     "ethernet-phy-ieee802.3-c22";
+		reg = <0x1>;
+		/*
+		 * LED[0] (Green): LINK/ACT
+		 * LED[1] (Amber): SPEED 100/1000M
+		 */
+		marvell,reg-init = <3 16 0 0x61>;
+	};
+};
+
+&cp0_ethernet {
+	status = "okay";
+};
+
+/* LAN1-5 (Switch) */
+&cp0_eth0 {
+	status = "okay";
+
+	phy-connection-type = "sgmii";
+	phys = <&cp0_comphy2 0>;
+
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+/* "WAN" */
+&cp0_eth1 {
+	status = "okay";
+
+	phy-connection-type = "sgmii";
+	phy-handle = <&ethphy1>;
+	phys = <&cp0_comphy0 1>;
+};
+
+&cp0_gpio1 {
+	board-id-1 {
+		gpio-hog;
+		gpios = <4 GPIO_ACTIVE_HIGH>;
+		input;
+	};
+
+	board-id-0 {
+		gpio-hog;
+		gpios = <10 GPIO_ACTIVE_HIGH>;
+		input;
+	};
+};
+
+&cp0_gpio2 {
+	board-id-2 {
+		gpio-hog;
+		gpios = <30 GPIO_ACTIVE_HIGH>;
+		input;
+	};
+};
+
+/* 2x NCT7802Y, EEPROM */
+&cp0_i2c0 {
+	status = "okay";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&pmx_cp0_i2c0_pins>;
+	clock-frequency = <100000>;
+
+	hwmon@28 {
+		compatible = "nuvoton,nct7802";
+		reg = <0x28>;
+	};
```

Comment:
I think LTD (Local Temperature Sensor) can be added as channel 0.
Please see nuvoton,nct7802.yaml kernel doc.

Reply:
It seems LTD already enabled without dts definitions. Sorry.